src/client/components/Button/Button.js

```diff
@@ -8,10 +8,11 @@ export const Button = (props) => {
     <button
       className={`button ${
         props.appearance === 'danger' ? 'button-danger' : ''
-      }`}
+      }${props.disabled === 'disabled' ? 'disabled-button' : ''}`}
```

Comment:
```suggestion
      } ${props.disabled === 'disabled' ? 'disabled-button' : ''}`}
```
You need a space here.